__tests__/test.js

```diff
@@ -305,6 +305,27 @@ it('works with nested types and inline fragments', () => {
     );
 });
 
+it('namedInlineFragments works with nested types and inline fragments', () => {
+    return testGetFields(
+        `
+    {
+        someType {
+            a
+            b
+            e {
+                ... on NestedType {
+                    x
+                }
+            }
+        }
+    }
+    `,
+        ['a', 'b', 'e.NestedType.x'],
```

Comment:
I'm not sure I like the representation here. This overloads the `.` to mean both actual nesting and this psedo-interface nesting. The query without the spread also produces the same result of `['a', 'b', 'e.NestedType.x']`.
```
{
    someType {
        a
        b
        NestedType {
            x
        }
    }
}
```

Reply:
Thank you for the feedback.
That's right, but IMHO not a bad behaviour. But if you'd like another join-element better I'm going to make the change. What do you think makes most sense? `>` / `_` `e>NestedType.x` / `e_NestedType.x`. Do you have a better idea? Both look very strange to me.